src/Infrastructure/TimeMgr/src/ESMCI_Alarm.C

```diff
@@ -438,6 +436,12 @@ int Alarm::count=0;
 
  } // end Alarm::set
 
+void Alarm::enableSticky(void){
+  // 1/100th of a second granuality to ensure integer return when being divided by
+  //this->ringInterval=TimeInterval(0, 1, 100, 0, 0, 0); 
+  this->ringInterval=TimeInterval(1, 0, 1, 0, 0, 0); 
+}
```

Comment:
- [ ] This warrants investigation: First, note that this is now using a 1 second granularity rather than the 1/100 second mentioned in the comment. But more importantly, it looks like this setting of `ringInterval` will differ depending on whether an alarm is initially created as sticky (in which case I don't think `enableSticky` is executed, and so ringInterval is either kept at what is set or set to a negative value for a one-shot), or if it's later set as sticky (in which case this `enableSticky` is called).